### PR TITLE
Add I2C pin definitions to motor2040 header

### PR DESCRIPTION
**Pull request by robberwick**
Add definitions for the QW/ST connector SDA & SCL pins

addresses #889

### CODE REVIEW

**Comment by ZodiusInfuser** [comment]
Thanks for this. As mentioned in issue #889 the intention was for people to access these pins via our I2C class, which has many convenience functions for dealing with I2C comms. I realise though that there is no harm in having these pins defined in `motor::MOTOR2040` for those wanting to roll their own solution though.

Before I approve this PR, could you amend the following comments.

Also, could you include the same three lines for SERVO2040 too please?

File: libraries/motor2040/motor2040.hpp [hunk 1]
```diff
@@ -49,6 +49,9 @@ namespace motor {
 
     const uint USER_SW = 23;
 
+    const uint I2C_SDA = 20;
```
Comment (ZodiusInfuser) [inline libraries/motor2040/motor2040.hpp:52]: Can these lines be put before USER_SW, so they are in number order?
Also `const uint I2C_INT = 19;` should be included too

Comment (robberwick) [reply libraries/motor2040/motor2040.hpp:51]: `I2C_INT` added and reordered in  [3e81b24](https://github.com/pimoroni/pimoroni-pico/pull/890/commits/3e81b245a1a16e3d791bf92210b822b52a7791ef)
`i2c_*` pin defs added to `servo2040.hpp` in [9e6a072](https://github.com/pimoroni/pimoroni-pico/pull/890/commits/9e6a0725c03daaaa8777a4e798c8c13fabf61000)

**Comment by robberwick** [comment]
@ZodiusInfuser Just checking this is ok, and there's nothing else you want me to do before it's mergeable?

**Comment by ZodiusInfuser** [comment]
Sorry for the delay. Yes, this is good. Thanks for the contribution!

**Pull request merged**